tsconfig.json

```diff
@@ -1,23 +1,16 @@
 {
   "compilerOptions": {
+    "baseUrl": "src",
+    "rootDir": "src",
+    "outDir": "build",
```

Comment:
To be consistent with the new interledgerjs mono repo and typescript convention we should use `dist`